src/configs.py

```diff
@@ -75,12 +75,26 @@ def load(cls, config: Mapping[str, Any]) -> "StartTLSConfig":
         )
 
 
+@dataclass
+class LdapOverTLSConfig:
+    enabled: bool = True
+    tls_key: Path = SERVER_KEY
+    tls_cert: Path = SERVER_CERT
+
+    @classmethod
+    def load(cls, config: Mapping[str, Any]) -> "LdapOverTLSConfig":
+        return StartTLSConfig(
+            enabled=config.get("ldap_over_tls_enabled", True),
```

Comment:
The default value in `charmcraft.yaml` is `false`, but here we set it to `True`. I think that the default value here will never be used (?), but let's keep it consistent.

Reply:
Fixed in 0a2e11b